packages/@react-spectrum/s2/src/Disclosure.tsx

```diff
@@ -195,7 +200,31 @@ const chevronStyles = style({
   flexShrink: 0
 });
 
+
 function DisclosureHeader(props: DisclosureHeaderProps, ref: DOMRef<HTMLDivElement>) {
+  let {
+    UNSAFE_className,
+    UNSAFE_style,
+    children
+  } = props;
+  let domRef = useDOMRef(ref);
+  return (
+    <div
+      style={UNSAFE_style}
+      className={(UNSAFE_className ?? '') + style({display: 'flex', alignItems: 'center', gap: 4})}
+      ref={domRef}>
+      {children}
```

Comment:
Do we need a context to disable, tshirt size, or quiet children?
<img width="262" alt="image" src="https://github.com/user-attachments/assets/f5226090-677b-4bf7-8049-ea7858dc1a9b">

Reply:
oh yes, good catch!